providers/httpnet/httpnetProvider.go

```diff
@@ -0,0 +1,179 @@
+package httpnet
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/StackExchange/dnscontrol/v3/models"
+	"github.com/StackExchange/dnscontrol/v3/pkg/diff"
+	"github.com/StackExchange/dnscontrol/v3/providers"
+)
+
+var defaultNameservers = []string{"ns1.routing.net", "ns2.routing.net", "ns3.routing.net"}
+
+var features = providers.DocumentationNotes{
+	providers.CanUseAlias:            providers.Can(),
```

Comment:
Please sort this list.

Reply:
I have sorted them according to [capabilities.go](https://github.com/StackExchange/dnscontrol/blob/c547beacc0bae0e16b4323abf74e2a7e00045045/providers/capabilities.go).

Should I order them alphabetically, instead?

Reply:
Just checked their capabilities again and they support SSHFP by now. Added it.

Reply:
Sorted them alphabetically now.